app/src/main/java/com/sopt/now/compose/presentation/ui/auth/component/AuthTextField.kt

```diff
@@ -0,0 +1,94 @@
+package com.sopt.now.compose.presentation.ui.auth.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.onFocusChanged
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.unit.dp
+import com.sopt.now.compose.R
+import com.sopt.now.compose.ui.theme.CustomTheme
+
+@Composable
+fun AuthTextField(
+    value: TextFieldValue,
+    onValueChange: (TextFieldValue) -> Unit,
+    modifier: Modifier = Modifier,
+    isFocused: Boolean,
+    onFocusChanged: (Boolean) -> Unit,
+    onRemove: () -> Unit,
+    isPassword: Boolean = false,
+    hint: String
+) {
+    Box(modifier = modifier) {
+        BasicTextField(
+            value = value,
+            onValueChange = { onValueChange(it) },
```

Comment:
```suggestion
            onValueChange = onValueChange,
```

제 기억상 이렇게만 해도 잘 돌아갈거에요 아마...

Reply:
앱잼때도 이거 고친적이 많은데 또 이렇게 해놓았네요 ㅋㅋㅋㅋ 근데 이게 없애도 될때가 있고 아닐때가 있어서 다시 한번 공부해봐야겠습니다..! 디테일 감사합니다!